styles.css

```diff
@@ -40,8 +40,10 @@ li {
 }
 
 .mobile-version-nav.active {
-  opacity: 1;
+  opacity: 0.8;
   pointer-events: auto;
+  filter: blur(8px);
+  -webkit-filter: blur(8px);
 }
```

Comment:
- [ ] Kindly your menu doesn't look very clear, you can remove `filter: blur(8px); & -webkit-filter: blur(8px);`. It will make it work.

![image](https://user-images.githubusercontent.com/35535130/168698870-d4a5f3b2-b5aa-4870-88d7-ae09cf31abe8.png)